editor/utils/formHelpers.ts

```diff
@@ -187,3 +188,13 @@ export const handleFormValueOutput = (
   // handle everything else as a nulllable string
   return trimStringNullable(value);
 };
+
+/**
+ * Generates validation rules for EditableField based on column definition
+ */
+export const commonValidations = {
+  isNumber: (value: string) => {
+    return /^\d+$/.test(value) || "This field must be a number";
+  },
+  required: "This field is required"
+};
```

Comment:
☝🏼

Reply:
looks like this file is missing a newline / linting

Reply:
thats what my pointer finger emoji was trying to point out